examples/berad/pkg/chain-spec/berachain.go

```diff
@@ -33,6 +33,9 @@ type BeraChainSpec struct {
 	]
 	// BGTContractAddress is the address of the BGT contract.
 	BGTContractAddress [20]byte `mapstructure:"bgt-contract-address"`
+	// BeraMintAmount is the amount of BERA minted per block to the BGT
+	// contract.
+	BeraMintAmount uint64 `mapstructure:"bera-mint-amount"`
```

Comment:
@calbera this is the example package.
Should we keep it up to date with prod chain specs?
I ask for the validator set cap PR as well